### ISSUE TITLE
Dropdown Menu when Extension Icon is clicked

### ISSUE THREAD

**Issue by patrickxchong** [patrickxchong/hide-discord-sidebar#4]
Options in dropdown menu:
- Enable/disable extension
- Enable/disable autohiding channels 
- Donate/Support button

I plan to keep the hide servers button in the main Discord UI for easy access

**Comment by DenverCoder1** [comment]
I just came across this extension and it's very useful! I might suggest to add an option to have the channels list collapse only when the window is narrow (i.e. show the channels list by default but start collapsing it when the window is less than 700px wide).

**Comment by patrickxchong** [comment]
Hey @DenverCoder1, thanks for the suggestion. I'll keep this in mind, might make it an advanced option since I won't want too many options in the main dropdown.

**Comment by patrickxchong** [comment]
Hello @DenverCoder1, I've implemented the feature here, would you be interested to be a beta tester for it?

**Comment by DenverCoder1** [comment]
@patrickxchong Looks good! I just checked out the V3 branch (https://github.com/patrickxchong/hide-discord-sidebar/tree/e20ed295f6cc6ffe8e1fecb3e2382c4f673cf1ac) and it looks like it works pretty well.

Here are a few thoughts on it:

1. Since there are two "small window width" settings buttons, I expected it to be possible to specify a different width for servers and channels, but instead changing one changes the other

2. Not a huge issue, but the radio buttons seem to be a little off center

3. The coffee button extends beyond the bottom of the popup, so maybe you could add more padding or make the popup longer

<details>
  <summary>Screenshot</summary>
	
![image](https://user-images.githubusercontent.com/20955511/119696818-3aede200-be58-11eb-95fa-84013b5c2c75.png)

</details>
	
In case there are display differences, the browser I used for testing is Brave 1.24.86 Chromium: 90.0.4430.212 (64-bit)

Great work! Definitely improves it! :+1:

**Comment by patrickxchong** [comment]
Hey @DenverCoder1, thank you so much for the feedback!

For 1, do you find that causes a lot of confusion? I wanted to put the gear button next to both options so that people know that they can change the Small Window Width setting there. Do you think it would help if I indicate that it's for both channels and servers as below? 
![image](https://user-images.githubusercontent.com/20418855/119850183-e2920f80-bf3f-11eb-80e8-d5cffe330c3e.png)


For 2 and 3, I think it's caused by browser rendering inconsistencies/user settings. Here's how it looks on on my end with Brave on Manjaro Linux. 
<details>
  <summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/20418855/119846562-d3f62900-bf3c-11eb-89eb-77f248c3a873.png)

</details>

I've updated the styles to be more robust. If it's not too much trouble, can you give the updated extension a try? Here's a zipfile that you can directly load into your browser: [hide-discord-sidebar-v3.zip](https://github.com/patrickxchong/hide-discord-sidebar/files/6555012/hide-discord-sidebar-v3.zip)

**Comment by DenverCoder1** [comment]
@patrickxchong I just installed the latest version and it looks good.

Adding the text that it is for servers and channels does help. It's possible that some people may miss it, but I think it is fine as it is.

The radio buttons are centered for me now, but the button is still pretty close to the edge. It does look much better, though.

![image](https://user-images.githubusercontent.com/20955511/119896618-ef1a6600-bf47-11eb-9602-1604cb7ca161.png)

**Comment by patrickxchong** [comment]
@DenverCoder1 I think I've fixed all possible UI issues, but since I couldn't reproduce the issue where the Buy Me a Coffee button is close to the edge, can you help with doing a final check for this new extension package? :pray: 
[hide-discord-sidebar-v3.zip](https://github.com/patrickxchong/hide-discord-sidebar/files/6564149/hide-discord-sidebar-v3.zip)

As always, appreciate your help!

**Comment by DenverCoder1** [comment]
This is (as before) using Brave on Arch Linux. It now has a scrollbar and there is space under the button if I scroll down.

![image](https://user-images.githubusercontent.com/20955511/120082733-40a02d80-c0cd-11eb-9ea6-080c213e2890.png)

![image](https://user-images.githubusercontent.com/20955511/120082795-8eb53100-c0cd-11eb-841d-707bf47ae4e8.png)

I think it looks fine. What's most important is that it's functional.

My Arch Linux configuration may be unique, so most users (using Mac or Windows) probably won't see the same thing I see. It should be enough to get it looking right on those operating systems (I'm not able to test on either of those right now, though).

**Comment by patrickxchong** [comment]
That's facinating, I'm running Manjaro and don't have the same issue. But it's okay, I added a little more space for the support text at the bottom, that should fix the scrollbar for you and anyone who may have this issue (hopefully!). The extension does look decent in my tests on Window and Mac, so hopefully that generalises to most people. 

I've uploaded the new extension package into the Chrome webstore and it'll be published once it's approved. Thank you so much for your feedback and support! :100: :star2:
